src/poker/game/ActionHandler.java

```diff
@@ -0,0 +1,165 @@
+        if (lastAction == PossibleAction.LIMP) {
+            hanldeLimp(activePlayer);
+        } else if (lastAction == PossibleAction.RAISE) {
+            handleRaisePlayer(activePlayer, nonActivePlayer);
+        } else if (lastAction == PossibleAction.BET) {
+            handleBetPlayer(activePlayer);
+        } else if (lastAction == PossibleAction.CALL) {
+            boolean isAllin = handleCall(activePlayer, nonActivePlayer);
+            result.setAllIn(isAllin);
+        } else if (lastAction == PossibleAction.CHECK) {
+            System.out.println(activePlayer.getNamePlayer() + " checks");
+        } else {
+            result.setFold(true);
+            handleFold(activePlayer, nonActivePlayer);
+        }
+        return result;
+    }
+
+
+    public void handleNextHand() {
+        for (int i = 0; i < 10; i++) {
+            System.out.println();
+        }
+        try {
+            Thread.sleep(4000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        this.result = deck.shuffleDeckAndGetHandsAndBoard();
+        this.gameState.setGamePhase(GamePhase.PREFLOP);
+        player1.setHandPlayer(result.getHand1());
+        player2.setHandPlayer(result.getHand2());
+    }
+
+
+    public boolean handleCall(Player activePlayer, Player nonActivePlayer) {
+        updatePotAndStackAmount(GamePlayUtils.amountToCall(nonActivePlayer, activePlayer), activePlayer);
+        return activePlayer.getStackPlayer() == 0 || nonActivePlayer.getStackPlayer() == 0;
+    }
+
+
+    public void handleNextStreet(GameState gameState) {
+        if (gameState.getGamePhase() == GamePhase.PREFLOP) {
+            GamePlayUtils.printPot(gameState);
+            gameState.setGamePhase(GamePhase.FLOP);
+            GamePlayUtils.printBoard(GamePhase.PREFLOP,result,gameState);
+        } else if (gameState.getGamePhase() == GamePhase.FLOP) {
+            GamePlayUtils.printPot(gameState);
+            gameState.setGamePhase(GamePhase.TURN);
+            GamePlayUtils.printBoard(GamePhase.FLOP,result, gameState);
+        } else if (gameState.getGamePhase() == GamePhase.TURN) {
+            GamePlayUtils.printPot(gameState);
+            gameState.setGamePhase(GamePhase.RIVER);
+            GamePlayUtils.printBoard(GamePhase.TURN,result, gameState);
+        } else {
+            resultAtShowdown();
+        }
+    }
+
+    public void hanldeLimp(Player activePlayer) {
+        System.out.println(activePlayer.getNamePlayer() + " limps 0,5$");
+        updatePotAndStackAmount(gameState.smallBlind, activePlayer);
+        activePlayer.increasePlayermoneyOnStreet(gameState.smallBlind);
+    }
+
+
+    public void handleRaisePlayer(Player activePlayer, Player nonActivePlayer) {
+        System.out.println("Podaj do ilu, raise minimum " + Math.max(2, 2 * nonActivePlayer.getPreviousRaiseOrBet()));
+        double raise = 0;
+        Scanner scanner = new Scanner(System.in);
```

Comment:
i wez zrob z tego scannera zmianna globalna, zeby nie robic we wszystkich funkcjach New bo to podejrzewam bedzie wolniejsze jak go zrobimy raz na poczatku i przypiszemy do zmiennej globalnej